### PR TITLE
(DOCSP-11033): Embedded Objects (Node/RN)

### PR DESCRIPTION
**Pull request by nlarew**
## Jira

- (DOCSP-11033): Embedded Objects (Node/RN)

## Staged Changes

Note: These pages are essentially duplicates

- [Embedded Objects (Node)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/embedded/node/embedded-objects.html)
- [Embedded Objects (React Native)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/embedded/react-native/embedded-objects.html)

### CODE REVIEW

File: source/node/embedded-objects.txt [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+object is deleted or if they are overwritten by a new embedded object.
+
+Embedded Object Data Models
+---------------------------
+
+You can define embedded object types using either Realm object models or a
+server-side document schema. Embedded object types are reusable and composable.
+You can use the same embedded object type in multiple parent object types and
+you can embed objects inside of other embedded objects. You can even recursively
+reference an embedded object type in its own definition.
```
Comment (cbush) [inline source/node/embedded-objects.txt:37]: Only if the recursive embedded object field is optional, I guess?

File: source/node/relationships.txt [hunk 2]
```diff
@@ -17,9 +17,9 @@ Overview
 
 {+service+} allows you to define explicit relationships between the types of
 objects in a {+app+}. A relationship is an object property that references
-another {+service-short+} object rather than one of the primitive data types. You
-define relationships by setting the type of an object property to
-another object type in the :ref:`property schema <node-property-schema>`.
+another {+service-short+} object rather than a scalar or embedded object. You
```
Comment (cbush) [inline source/node/relationships.txt:20]: Should this change go in the corresponding RN file as well?

File: source/node/embedded-objects.txt (hunk 1 above)
Comment (nlarew) [reply source/node/embedded-objects.txt:37]: Hmm interesting I'll follow up

File: source/node/relationships.txt (hunk 2 above)
Comment (nlarew) [reply source/node/relationships.txt:20]: Yes good catch!

**Pull request merged**